docs/pages/server-access/guides/openssh/openssh-manual-install.mdx

```diff
@@ -279,7 +279,6 @@ $ tsh status
   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
 ```
 
-</ScopedBlock>
 </ScopedBlock>
```

Comment:
The docs engine no longer supports these, so we'll need to replace them with either:
- `Tabs` components if there are significant variations for different scopes
- Body text for the most-seen variation. If it makes sense, we could use `Var`s for values that a user will need to modify.

Reply:
I copied what's currently being done in the current docs, using tabs as I'm not sure what's used the most. Is that fine?

Reply:
Using `Tabs` is fine for now! We'll just need to not use `ScopedBlock`s because the docs engine no longer supports them.

Reply:
Just realized there were some `ScopedBlock`s I missed, all should be gone now